### ISSUE TITLE
Multisig Actor Test Harness verifies exitcode.

### ISSUE THREAD

**Issue by ZenGround0** [filecoin-project/specs-actors#471]
From builtin/multisig/multisig_test.go:

```golang
// TODO In a follow-up, this method should also verify the return value from Approve contains the exit code prescribed in ExpectSend.
// exercise both un/successful sends.
func (h *msActorHarness) approve(rt *mock.Runtime, txnID int64, proposalParams []byte) {
```

**Comment by frrist** [comment]
Attempting to address this as apart of https://github.com/filecoin-project/specs-actors/pull/434/files#diff-d5777d603b48d25386edace2468db7cfL1100

**Comment by ZenGround0** [comment]
Yeah literally just saw this, thanks @frrist.  We'll close this one down when that one lands.

**Comment by anorth** [comment]
Closed by #434
